pkg/repo/mysql/mysql.go

```diff
@@ -0,0 +1,34 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	_ "github.com/go-sql-driver/mysql"
+)
+
+type Conf struct {
+	Addr     string
+	User     string
+	Password string
+	Database string
+}
+
+func DefaultConf() (*Conf, error) {
+	return new(Conf), nil
+}
+
+func NewDB(ctx context.Context, conf *Conf) (*sql.DB, error) {
+	db, err := sql.Open("mysql", fmt.Sprintf("%s:%s@tcp(%s)/%s", conf.User, conf.Password, conf.Addr, conf.Database))
```

Comment:
There should be OTEL wrapper for sql.Open.

Reply:
Found 3 packages, this is the one I liked the most https://github.com/XSAM/otelsql
Maybe in the future we will need to do manual instrumenting, because all those packages are not super configurable